packages/discord.js/src/client/actions/InteractionCreate.js

```diff
@@ -29,6 +31,7 @@ class InteractionCreateAction extends Action {
             InteractionClass = UserContextMenuCommandInteraction;
             break;
           case ApplicationCommandType.Message:
+            if (c && !c.isTextBased()) return;
```

Comment:
```suggestion
            if (!c?.isTextBased()) return;
```

Shouldn't this also work?

Reply:
No. This will break functionality for applications without bots.

Reply:
I mean, why can't we use an optional chaining operator?